docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/search_tool_characteristics.rst

```diff
@@ -0,0 +1,51 @@
+Search Tool Characteristics
+===========================
+
+The S3 Search tool is a Scality API extension to the AWS API. S3 Search
+complies with AWS S3 search syntax, with certain noteworthy differences. S3
+Search is MongoDB-native, and addresses the S3 search through queries
+encapsulated in a SQL WHERE predicate. It uses Perl-Compatible Regular
+Expression (PCRE) search syntax.
+
+Searchable Metadata and Tags
+----------------------------
+
+Because |product| supports two classes of custom tags, it is not possible to
+list all the ways to search data in |product|. At a mimimum, however, files will
+have the following fields inscribed, and these will be searchable:
+
+-  cache-control
```

Comment:
hmm, the fact that some are kebab cased and some are camel-cased is a little lame.

Reply:
Not my department... I just tell it like it is.